README.md

```diff
@@ -311,6 +327,8 @@ $ aws events put-targets --rule MyK8sScheduledChangeRule \
   --targets "Id"="1","Arn"="arn:aws:sqs:us-east-1:123456789012:MyK8sTermQueue"
 ```
 
+You may skip this step if sending events from ASG to SQS directly.
```

Comment:
Love the reordered steps, they make the setup process much more clear!

This sentence would be better at the beginning of step 4, so the customer doesn't have to read any more than necessary.